demo/llm.vdb.service/createvdb.yaml

```diff
@@ -36,8 +36,8 @@ spec:
           - name: VECTOR_DB_S3_FILE
             value: ${VECTOR_DB_S3_FILE}
           - name: EMBEDDING_CHUNK_SIZE
-            value: ${EMBEDDING_CHUNK_SIZE}
+            value: "${EMBEDDING_CHUNK_SIZE}"
           - name: EMBEDDING_CHUNK_OVERLAP
-            value: ${EMBEDDING_CHUNK_OVERLAP}
+            value: "${EMBEDDING_CHUNK_OVERLAP}"
           - name: EMBEDDING_MODEL_NAME
             value: ${EMBEDDING_MODEL_NAME}
```

Comment:
Using pydantic settings introduces one impediment. Before running 
```
envsubst < createvdb.yaml | kubectl apply -f -
```

We must make sure all env variables are exported. If for ex `EMBEDDING_CHUNK_SIZE` is not set in our terminal, envsubst will put an empty string there and pydantic settings will complain that they can't change empty setting to integer.

I will dig deeper into pydantic settings and try to make sure that even in such a case we will use defaults.